app/src/main/java/com/example/sabonit/ProductActivity.java

```diff
@@ -33,6 +34,8 @@
 
 public class ProductActivity extends AppCompatActivity {
 
+    private final int NEW_BOTTLE = -1;
```

Comment:
some documentation is needed...

Reply:
right. actually the value is random, we use it only because it cant be negative num of liters